docs/content/querying/_index.md

```diff
@@ -90,39 +94,98 @@ method `toSet` is called. The resulting Scala set is converted to a
 list and sorted. Note that both `toList` and `sorted` are API methods
 of the host language Scala.
 
-The **id** is a long integer that uniquely identifies the node. It can
-be used to jump to the node in subsequent queries. For example, you
-can determine the ids of all File nodes as follows:
+[1] The term `Step` is inherited from the graph traversal language
+[Gremlin](http://tinkerpop.apache.org/docs/current/reference/#graph-traversal-steps),
+and more specifically
+[gremlin-scala](https://github.com/mpollmeier/gremlin-scala) on which
+our DSL is based. In Gremlin, `Steps` describe walks in a graph, which
+are combined via function composition to yield *traversals*. You can
+think of traversals as expressions that describe sets of nodes.
+
+
+
+## Expression Types and Starters
+
+For each node label, a corresponding *expression type* is defined in the
```

Comment:
I'm not 100% sure if this change makes sense, but just calling it a `type` seems more intuitive to me.

```suggestion
For each node label, a corresponding *type* is defined in the
```